ember-cli/2016-12/december-01.md

```diff
@@ -0,0 +1,79 @@
+# ember-cli Meeting Notes - 2016-12-01
+
+## Attendees
+
+- Nathan Hammond
+- Jake Bixby
+- Tobias Bieniek
+- Alex Navasardyan
+- Trent Willis
+- Gavin Joyce
+- Kelly Selden
+- Katie Gengler
+- Chad Hietala
+- David Hamilton
+- Robert Jackson
+- Miguel Madero
+ 
+## Actions
+
+- [x] Nathan - Land RFC #50.
+- [x] Nathan - Get access to ember-cli npm user from Stef.
+- [x] Nathan - [Respond on `ember-cli-release`](https://github.com/ember-cli/ember-cli/issues/6462#issuecomment-264322357)
+ 
+## Discussion Notes
+
+### [RFC #50 Production Code Stripping](https://github.com/ember-cli/rfcs/pull/50)
+
+- Given no additional comments in the FCP, this was merged.
+- Excited to make it happen.
+- Thanks to @gavinjoyce for bringing it home!
+ 
+### 2.11.0-beta.1 Postmortem
+
+- CI was broken for two weeks leading up to the 2.11.0-beta.1 release.
+- [This is the fix to what broke us.](https://github.com/stefanpenner/capture-exit/pull/7)
+  - We threw away the exit code because it was an async exit, but you can't run
+    async code on top-level exit.
```

Comment:
This is not quite correct.

We threw away the exit code because we tried to defer exiting after async work during `process.on('exit')`, in which async work is explicitly disallowed.


You actually *can* run async on top-level exit, and we've added support for this in ember-cli/capture-exit#7